src/EditorFeatures/CSharpTest/Diagnostics/Nullable/CSharpDeclareAsNullableCodeFixTests.cs

```diff
@@ -969,5 +969,36 @@ class C
 }",
                 parameters: s_nullableFeature);
         }
+
+        [Fact, WorkItem(44983, "https://github.com/dotnet/roslyn/issues/44983")]
+        public async Task FixFieldDeclaration_Unassigned()
+        {
+            await TestInRegularAndScript1Async(
+@"#nullable enable
+
+class C
+{
+    private string [|_value|];
+}",
+@"#nullable enable
+
+class C
+{
+    private string? _value;
+}",
+                parameters: s_nullableFeature);
+        }
+
+        [Fact]
+        [WorkItem(44983, "https://github.com/dotnet/roslyn/issues/44983")]
+        public async Task MultipleDeclarator_NoDiagnostic()
+        {
+            await TestMissingInRegularAndScriptAsync(
+@"#nullable enable
+class Program
+{
+    string s, s2 = ""hello"";
```

Comment:
```suggestion
    string [|s|], s2 = ""hello"";
```

TestMissing... still takes a span where you're asserting that the fix doesn't apply.

Reply:
@jasonmalinowski  Thanks.
I thought the span applies only to the tests that has diagnostics.